src/process/loader-shell-runner.test.ts

```diff
@@ -0,0 +1,3 @@
+Deno.test("unregistering a signal handler that was never registered has no bad side-effects", () => {
+  Deno.removeSignalListener("SIGTERM", () => {});
```

Comment:
d: I guess you dont need to test for any conditions here? Just remove it and that is basically the test?

Reply:
yip